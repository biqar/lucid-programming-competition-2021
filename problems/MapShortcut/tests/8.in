10
Right 163 17
Left 115 15
Right 46 9
Right 1 7
Left 50 3
Right 166 6
Right 52 11
Left 8 9
Right 62 2
Left 103 15
