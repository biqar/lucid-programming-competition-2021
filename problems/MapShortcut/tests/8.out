Right 176 25
